frontend/src/components/form/Form.tsx

```diff
@@ -0,0 +1,159 @@
+import {
+  Form as AForm, Input, Row, Typography,
+} from 'antd';
+import styled from 'styled-components';
+
+import { Button } from '@/components/buttons';
+import { Colors } from '@/styles/colors';
+
+const StyledForm = styled(AForm)`
+  background: ${Colors.onPrimary};
+  border: 1px solid ${Colors.shadow};
+  padding: 20px;
+  border-radius: 10px;
+
+  .first-and-last-name-form {
+      display: flex;
+      justify-content: space-between;
+      gap: 30px;
+  }
+
+  .first-and-last-name-form > * {
+    width: 100%;
+  }
+
+  .first-and-last-name-form Input {
+    width: 100%;
+  }
+
+  .form-item.label {
+    border: 1px blue solid;
+  }
+
+  .submit-btn-parent {
+    width: 100%;
+  }
+
+  Input {
+    border-radius: 5px;
+  }
+`;
+
+const Form: React.VFC = () => {
+  const [form] = StyledForm.useForm();
+  const { Title } = Typography;
+  return (
+    <div>
+      <Title>Adding New User</Title>
+      <StyledForm
+        form={form}
+        name="add-new-user"
+        layout="vertical"
+      >
+        <div className="first-and-last-name-form">
+          <StyledForm.Item
+            className="form-item"
+            label="First name"
+            name="firstname"
+            required
+            tooltip="Required field"
+            rules={[
+              {
+                required: true,
+                message: 'Please enter your first name!',
+                whitespace: false,
+                max: 256,
+              },
+            ]}
+          >
+            <Input placeholder="First Name" />
+          </StyledForm.Item>
+          <StyledForm.Item
+            className="form-item"
+            label="Last name"
+            name="lastname"
+            required
+            tooltip="Required field"
+            rules={[
+              {
+                required: true,
+                message: 'Please enter your last name!',
```

Comment:
Tak swoją drogą `!` oznacza, że na kogoś krzyczysz 😄 Użytkownicy odbierają to jako albo rozkaz albo pogardę, ale generalnie oba są źle nacechowane. Wykrzyknik jest stosowany tylko w niektórych sytuacjach, w których chcemy coś mocno podkreślić, np. `Witaj drogi użytkowniku!`.

Ale w tych naszych wiadomościach wystarczy `Proszę wpisać/podać nazwisko`